[2024-12-02T18:45:37.345+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-02T18:45:37.369+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_kafka_dag.store_openmeteo_data manual__2024-12-02T18:45:01.571504+00:00 [queued]>
[2024-12-02T18:45:37.376+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_kafka_dag.store_openmeteo_data manual__2024-12-02T18:45:01.571504+00:00 [queued]>
[2024-12-02T18:45:37.378+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 4
[2024-12-02T18:45:37.390+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): store_openmeteo_data> on 2024-12-02 18:45:01.571504+00:00
[2024-12-02T18:45:37.397+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=15011) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-02T18:45:37.398+0000] {standard_task_runner.py:64} INFO - Started process 15014 to run task
[2024-12-02T18:45:37.397+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_data_kafka_dag', 'store_openmeteo_data', 'manual__2024-12-02T18:45:01.571504+00:00', '--job-id', '613', '--raw', '--subdir', 'DAGS_FOLDER/kafka_stream.py', '--cfg-path', '/tmp/tmpcrt8mliw']
[2024-12-02T18:45:37.399+0000] {standard_task_runner.py:91} INFO - Job 613: Subtask store_openmeteo_data
[2024-12-02T18:45:37.437+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_data_kafka_dag.store_openmeteo_data manual__2024-12-02T18:45:01.571504+00:00 [running]> on host 0741a2a1545f
[2024-12-02T18:45:37.518+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='weather_data_kafka_dag' AIRFLOW_CTX_TASK_ID='store_openmeteo_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-02T18:45:01.571504+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-02T18:45:01.571504+00:00'
[2024-12-02T18:45:37.519+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-02T18:45:37.532+0000] {conn.py:362} INFO - <BrokerConnection node_id=bootstrap-0 host=broker:9092 <connecting> [IPv4 ('172.18.0.11', 9092)]>: connecting to broker:9092 [('172.18.0.11', 9092) IPv4]
[2024-12-02T18:45:37.532+0000] {conn.py:957} INFO - Probing node bootstrap-0 broker version
[2024-12-02T18:45:37.533+0000] {conn.py:393} INFO - <BrokerConnection node_id=bootstrap-0 host=broker:9092 <connecting> [IPv4 ('172.18.0.11', 9092)]>: Connection complete.
[2024-12-02T18:45:37.637+0000] {conn.py:1019} INFO - Broker version identified as 2.6.0
[2024-12-02T18:45:37.638+0000] {conn.py:1020} INFO - Set configuration api_version=(2, 6, 0) to skip auto check_version requests on startup
[2024-12-02T18:45:37.642+0000] {consumer.py:122} WARNING - group_id is None: disabling auto-commit.
[2024-12-02T18:45:37.643+0000] {subscription_state.py:167} INFO - Updating subscribed topics to: ('openmeteo_raw',)
[2024-12-02T18:45:37.646+0000] {subscription_state.py:253} INFO - Updated partition assignment: [('openmeteo_raw', 0)]
[2024-12-02T18:45:37.649+0000] {conn.py:362} INFO - <BrokerConnection node_id=1 host=broker:9092 <connecting> [IPv4 ('172.18.0.11', 9092)]>: connecting to broker:9092 [('172.18.0.11', 9092) IPv4]
[2024-12-02T18:45:37.650+0000] {conn.py:393} INFO - <BrokerConnection node_id=1 host=broker:9092 <connecting> [IPv4 ('172.18.0.11', 9092)]>: Connection complete.
[2024-12-02T18:45:37.651+0000] {conn.py:673} INFO - <BrokerConnection node_id=bootstrap-0 host=broker:9092 <connected> [IPv4 ('172.18.0.11', 9092)]>: Closing connection. 
[2024-12-02T18:45:37.767+0000] {logging_mixin.py:188} INFO - Consumed 37 records from topic openmeteo_raw
[2024-12-02T18:45:37.785+0000] {logging_mixin.py:188} INFO - Deleted existing file at /opt/***/data/parquet_files/openmeteo_data.parquet
[2024-12-02T18:45:37.850+0000] {logging_mixin.py:188} INFO - Data from openmeteo_raw stored in /opt/***/data/parquet_files/openmeteo_data.parquet
[2024-12-02T18:45:37.851+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-12-02T18:45:37.852+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-02T18:45:37.858+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_data_kafka_dag, task_id=store_openmeteo_data, run_id=manual__2024-12-02T18:45:01.571504+00:00, execution_date=20241202T184501, start_date=20241202T184537, end_date=20241202T184537
[2024-12-02T18:45:37.894+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-12-02T18:45:37.907+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-02T18:45:37.910+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
